### ISSUE TITLE
should flag focusable elements inside a scroll region with height 0

### ISSUE THREAD

**Issue by straker** [dequelabs/axe-core#2263]
Focusable elements inside a scrollable region with 0 height should be flag as violations of Success Criterion 2.4.7: Focus Visible.

```html
<div style="height: 0; overflow-y: scroll">
  <a href="...">link</a>
</div>
```

A secondary issue might be if a scrollable region with focusable content is less than 44px (height or width) we flag as a violation of Success Criterion 2.5.5 Target Size.

**Comment by dylanb** [comment]
also width 0?

**Comment by WilcoFiers** [comment]
This feels a little error prone. Going to close this.
